public/js/thirdpartyservices.js

```diff
@@ -102,5 +102,15 @@ var thirdPartyServicesJsonArray = [
       privacy: 'https://privacy.oath.com',
       terms: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html'
     }
+      , // Donot forget to add a ',' before adding your new item in the array
```

Comment:
Can you move the comment to the last line, because it is a reminder for others who submit PR

Reply:
Done